sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormTrainingClient.cs

```diff
@@ -111,8 +111,11 @@ public FormTrainingClient(Uri endpoint, TokenCredential credential, FormRecogniz
         /// <param name="useTrainingLabels">If <c>true</c>, use a label file created in the &lt;link-to-label-tool-doc&gt; to provide training-time labels for training a model. If <c>false</c>, the model will be trained from forms only.</param>
         /// <param name="trainingFileFilter">Filter to apply to the documents in the source path for training.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
-        /// <returns>A <see cref="TrainingOperation"/> to wait on this long-running operation.  Its <see cref="TrainingOperation"/>.Value upon successful
-        /// completion will contain meta-data about the trained model.</returns>
+        /// <returns>
+        /// <para>A <see cref="TrainingOperation"/> to wait on this long-running operation. Its <see cref="TrainingOperation"/>.Value upon successful
+        /// completion will contain meta-data about the trained model.</para>
+        /// <para>If the training fails, an exception is raised, and a model with an "invalid" status is still created in the Form Recognizer resource.</para>
```

Comment:
Could we reword this to say something like:

- Even if training fails, a model is created in the Form Recognizer account with an "invalid" status.  A [type of]Exception will be raised containing the modelId to access this invalid model.